### ISSUE TITLE
Custom frametime limit is ignored

### ISSUE THREAD

**Issue by flibberdipper** [cirquit/trdrop#137]
Just started trying to use this and noticed that it outright ignores the custom value I set for the frametime graph. I have it set to 90, but as soon as it goes beyond that it automatically adjusts the graph to whatever the highest spike was.

EDIT: Also noticed the sidebar option to modify further frametime settings is grayed out, so maybe I'm just jumping the gun and this feature still isn't fully complete?

**Comment by illusion0001** [comment]
Please try #137

**Comment by illusionyy** [comment]
Please try #139

**Comment by Rossa-GH** [comment]
Was messing around with trdrop a while ago and noticed this to be an issue.

Did a quick test with that new build and it seems to fix it. Only bit of weirdness is the line can now go out of the limits of the graph (see example below). 

I know that the alternative is putting a hard cap on the line and you then don't see the true nature of the spike, but it would make things look cleaner though.

![Untitled_2 1 1](https://user-images.githubusercontent.com/10433269/194193335-f5db9b5a-5caf-4d58-a4ef-65a4a80eedb3.jpg)

**Comment by illusionyy** [comment]
Yea it's a double edge sword situation.

**Comment by Rossa-GH** [comment]
Yeah, it definitely is. Personally I'd prefer the hard limit, but that's just me.

Ideally there'd probably be a UI toggle between the OG version and this, but with hard limit applied to the line.

You guys are focused on v2 though, so I know adding features like that is probably not worth the time at this point. Either way, it's a good update, thanks.

**Comment by Rossa-GH** [comment]
Also, I feel like it's worth mentioning that the changes this problem causes are not permanent. The graph eventually reverts to the original max value, but it waits for the spike to be long gone. (Example below)

Maybe a quicker way to mitigate this issue would be to make the graph revert to the original max value the second the spike leaves the graph?


These 2 screenshots are exactly 20 second apart. Analysis range is 30 frames.

![Untitled_2 1 1](https://user-images.githubusercontent.com/10433269/194219813-74f6adaf-587e-4126-8d8e-8d9d97a4d9cf.jpg)
![Untitled_2 1 2](https://user-images.githubusercontent.com/10433269/194219820-a4978030-6e02-4da3-8ea6-107d8ee5622f.jpg)
